src/Gateway/Events/GuildMembersChunk.php

```diff
@@ -12,7 +15,7 @@ class GuildMembersChunk
     public string $guild_id;
 
     /**
-     * @var \Ragnarok\Fenrir\Parts\GuildMember[]
+     * @var GuildMember[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)